core/src/scss/elements/input/_select.scss

```diff
@@ -0,0 +1,14 @@
+@charset "UTF-8";
+
+// Select dropdown
+
+select {
+  @extend %input-base;
+  outline: none;
```

Comment:
Does this affect the focus state? I don't think we want to remove the outline on the focus of any input without handling it a different way for visually impaired users.

Reply:
I actually would vote for putting the outline back. However, it was in the original input.scss and might be used by some sites so I kept the `outline: none;` but took it out of the placeholder. What do you think? Should I just put the outline back into everything? @JBCSU  please feel free to comment too.

Reply:
> Does this affect the focus state? I don't think we want to remove the outline on the focus of any input without handling it a different way for visually impaired users.

And yes, it does affect the focus state. The current Decanter style remove all outline for focus state, and gives them a drop shadow instead which I'm not crazy about either.